R/probability_estimators.R

```diff
@@ -344,64 +482,81 @@ johnson_method_ <- function(data) {
 }
 
 
+
 #' Estimation of Failure Probabilities using Kaplan-Meier
 #'
 #' @description
 #' \lifecycle{soft-deprecated}
 #'
-#' The functionality of this function is incorporated into \code{\link{estimate_cdf}}.
+#' \code{kaplan_method()} is no longer under active development, switching
+#' to \code{\link{estimate_cdf}} is recommended.
 #'
 #' @details
 #' Whereas the non-parametric Kaplan-Meier estimator is used to estimate the
 #' survival function \emph{S(t)} in terms of (multiple) right censored data, the
 #' complement is an estimate of the cumulative distribution function \emph{F(t)}.
 #' One modification is made in contrast to the orginial Kaplan-Meier estimator
-#' (based on \emph{NIST/SEMATECH e-Handbook of Statistical Methods}, 8.2.1.5.):
-#' If the last unit (unit with highest observed lifetime) is a defective unit,
-#' the estimator is adjusted in such a way that the survival estimate for this
-#' unit is not \emph{zero} and therefore the estimate for the failure probability is
-#' not equal to \emph{one}. Otherwise the estimate in this context would be too
-#' pessimisitc.
-#' Since the failure probability estimation in this function is not based on
-#' \emph{Median Ranks}, the Betabinomial confidence intervals cannot be
-#' calculated on the basis of Kaplan-Meier failure probabilities.
+#' (see 'References').
+#'
+#' \strong{Note}: The \emph{Kaplan-Meier} estimator does not assign ranks to
```

Comment:
Maybe file an issue that confint_betabinom should throw a warning if method is "kaplan". With the new interface (`confint_betabinom.model_estimation`) we know the method.